docs/notebooks/index.md

```diff
@@ -14,47 +14,49 @@ jupyter:
 
 # PDO Contracts Launch Page #
 
-
-[Hyperledger Private Data Objects](https://github.com/hyperledger-labs/private-data-objects) operates as a Hyperledger Labs project. This code is provided solely to demonstrate basic PDO mechanisms and to facilitate
-collaboration to refine PDO architecture and define minimum viable product requirements.
+[Hyperledger Private Data Objects](https://github.com/hyperledger-labs/private-data-objects)
+operates as a Hyperledger Labs project. This code is provided solely
+to demonstrate basic PDO mechanisms and to facilitate collaboration to
+refine PDO architecture and define minimum viable product
+requirements.
 
 **The PDO contracts code provided here is prototype code and not intended for production use.**
 
-
 ## Getting Started
 
-
-There are a number of configuration options for interacting with PDO contracts. More information can be found in the [Getting Started](documents/getting_started.ipynb) notebook.
-
-
-
+There are a number of configuration options for interacting with PDO
+contracts. More information can be found in the
+[Getting Started](common/documents/getting_started.ipynb) notebook.
 
 ## Contract Families
 
-
 ### Exchange
 
+The Exchange contract family is a suite of contracts that demonstrate
+many of the capabilities of the private data objects
+technologies. Three basic contracts define the elements of the
+Exchange contract family: the asset type contract, the vetting
+organization contract, and the issuer contract. Three additional
+contracts extend the Exchange contract family for trading potentially
+confidential assets using non-fungible tokens.
 
-The Exchange contract family is a suite of contracts that demonstrate many of the capabilities of the private data objects technologies. Three basic contracts define the elements of the Exchange contract family: the asset type contract, the vetting organization contract, and the issuer contract. Three additional contracts extend the Exchange contract family for trading potentially confidential assets using non-fungible tokens.
-
-To experiment with these contracts, explore the [Exchange Contract Family notebook](exchange/index.ipynb)
-
-
+To experiment with these contracts, explore the
+[Exchange Contract Family notebook](exchange/index.ipynb)
```

Comment:
same as above. these are designed to work from the jupyter server, not from the shell. this needs to be clarified. AND... per your more general comment, it would be nice to build a dev image that had a jupyter server running so you could get at this information structured correctly.

Reply:
ahh cool! so I suggest that somewhere, maybe the root README.md one should explain how jump into the dev container and how to read things using jupyter. However, if the "jupyter" root is set to the repository root, couldnt one create links that work for both, jupyter and the normal github or any local IDE? 🤓

Reply:
possibly... 

first i'm going to 1) create a getting started container, 2) update the readme with instructions on how to get that started, and possibly 3) make the notebooks here more interactive & supportive of the builds.

we do have the issue that many of the docs are generated by jupytext (jupyter notebooks are notoriously difficult to manage with git).